src/components/pages/Navbar/NavbarFeatures/LoginButton.js

```diff
@@ -1,9 +1,25 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+
 import { useAuth0 } from '@auth0/auth0-react';
 import { NavBtnLink } from '../../NavBarLanding/NavBarStyle';
 
 function LoginButton() {
-  const { loginWithRedirect } = useAuth0();
+  const { loginWithRedirect, getAccessTokenSilently } = useAuth0();
+
+  useEffect(() => {
+    const getUserData = async () => {
+      try {
+        const token = await getAccessTokenSilently({
+          audience: 'underdog-identifier',
+          scope: 'openid profile email',
+        });
+        localStorage.setItem('AuthToken', token);
+      } catch (e) {
+        console.log(e.message);
```

Comment:
let's handle the error here.

Reply:
Could we meet to discuss what you had in mind?